[2025-05-12T08:00:28.023+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transformation_pipeline.transform_data scheduled__2025-05-11T08:00:08.175555+00:00 [queued]>
[2025-05-12T08:00:28.032+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transformation_pipeline.transform_data scheduled__2025-05-11T08:00:08.175555+00:00 [queued]>
[2025-05-12T08:00:28.033+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-05-12T08:00:28.048+0000] {taskinstance.py:1380} INFO - Executing <Task(DockerOperator): transform_data> on 2025-05-11 08:00:08.175555+00:00
[2025-05-12T08:00:28.052+0000] {standard_task_runner.py:57} INFO - Started process 66 to run task
[2025-05-12T08:00:28.056+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_transformation_pipeline', 'transform_data', 'scheduled__2025-05-11T08:00:08.175555+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/transformation_dag.py', '--cfg-path', '/tmp/tmpylif5qbg']
[2025-05-12T08:00:28.058+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask transform_data
[2025-05-12T08:00:28.075+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 21 for task transform_data (invalid interpolation syntax in 'dag_id={{ ti.dag_id }}/run_id={{ ti.run_id }}/task_id={{ ti.task_id }}/{% if ti.map_index >= 0 %}map_index={{ ti.map_index }}/{% endif %}attempt={{ try_number }}.log' at position 72; 66)
[2025-05-12T08:00:28.086+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-12T08:00:28.103+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
